Release a new version of Az.StackHCI module (#22722)
Azure/azure-powershell: src/StackHCI/docs/Get-AzStackHCILogsDirectory.md

---
external help file:
Module Name: Az.StackHCI
online version: https://learn.microsoft.com/powershell/module/az.stackhci/get-azstackhcilogsdirectory
schema: 2.0.0
---

# Get-AzStackHCILogsDirectory

## SYNOPSIS
Returns Logs directory path on the current node.

## SYNTAX

```
Get-AzStackHCILogsDirectory [[-Credential] <PSCredential>] [[-ComputerName] <String>] [-Confirm] [-WhatIf]
 [<CommonParameters>]
```

## DESCRIPTION
Returns Logs directory path on the current node.

## EXAMPLES

> ### -------------------------- EXAMPLE 1 --------------------------
> ```powershell
> The example below returns the logs directory path on the current node.
> ```
> 
> C:\PS\> Get-AzStackHCILogsDirectory
> HCI Registration Logs directory path: C:\ProgramData\AzureStackHCI

Reviewer: ```suggestion
### Example 1: Return the logs directory path on the current node.
```powershell
Get-AzStackHCILogsDirectory
```

```
